src/everett/manager.py

```diff
@@ -64,28 +60,6 @@ def qualname(thing):
     if inspect.ismodule(thing):
         return ".".join(parts)
 
-    # If it's a class
-    if inspect.isclass(thing):
-        parts.append(thing.__name__)
-        return ".".join(parts)
-
-    # If it's a function
-    if isinstance(thing, (types.FunctionType, types.MethodType)):
-        # If it's a method or function
-        if inspect.ismethod(thing):
-            if thing.im_class is type:
-                # This is a class method
-                parts.append(thing.im_self.__name__)
-            else:
-                # This is an bound/instance method
-                parts.append(thing.im_class.__name__)
-            parts.append(thing.__name__)
-
-        elif inspect.isfunction(thing):
-            parts.append(thing.__name__)
-
-        return ".".join(parts)
```

Comment:
Functions and methods have `__qualname__` in Python 3.3 and later. The code here uses `im_class` and `im_self` which is a relic of Python 2.